tests/search/vespa/fixtures/vespa_test_schema/schemas/document_passage.sd

```diff
@@ -54,13 +54,17 @@ schema document_passage {
     import field family_document_ref.family_publication_ts as family_publication_ts {}
     import field family_document_ref.family_publication_year as family_publication_year {}
     import field family_document_ref.family_geography as family_geography {}
+    import field family_document_ref.family_geographies as family_geographies {}
     import field family_document_ref.family_source as family_source {}
     import field family_document_ref.document_import_id as document_import_id {}
     import field family_document_ref.document_slug as document_slug {}
     import field family_document_ref.document_languages as document_languages {}
     import field family_document_ref.document_content_type as document_content_type {}
     import field family_document_ref.document_cdn_object as document_cdn_object {}
     import field family_document_ref.document_source_url as document_source_url {}
+    import field family_document_ref.corpus_import_id as corpus_import_id {}
```

Comment:
Just for me (for context), what effect(s) do adding new fields into this fixture file have?

Reply:
Yeah thanks for asking! So this defines the passages schema object in vespa (think the equivalant of an rds table). They existed on the document object but the root cause of the filtering problem is basically they got missed from passages. We already updated the live instances as well as the test instance in the sdk. So this updates the test instance schema here too